rules.ini

```
@@ -185,6 +185,8 @@ Anzu = (?:^|/)anzu\.dll$
 Azure_Playfab_Party = (?:^|/)Party(?:Win|Win7|Win32|XboxLive|XboxLiveWin32)\.dll$
 BASS = (?:^|/)(?:bass(?:flac|midi)?\.dll|libbass\.(?:dylib|so))$
 Bink_Video = (?:^|/)bink2?w(?:64|32)?\.dll$
+Box2D[] = (?:^|/)box2d\.(?:(?:XNA\.)?dll|txt|wasm(?:\.js)?)$
+Box2D[] = (?:^|/)libs/gdx-box2d
```

```suggestion
Box2D[] = (?:^|/)box2d\.(?:(?:XNA\.)?dll|txt|wasm)$
```
second one is for one game only so won't add